ui_panels.py

```diff
@@ -2324,7 +2324,10 @@ def draw_thumbnail_box(self, layout, width=250):
             op.tooltip = f"Complexity, median from {rc['workingHours']} rating{'' if rc['workingHours'] == 1 else 's'}" \
                          f"{tooltip_extension if rcount <= show_rating_threshold else ''}"
 
-            if rcount <= show_rating_prompt_threshold:
+            if rcount <= show_rating_prompt_threshold and \
```

Comment:
👍

Reply:
I think the reasons are different - that is, this behaviour was already there and it's a known bug, just now I didn't have the capacity to resolve it - since it needs a new call to api, to delete the rating. I'd merge this and then resolve the other issue in next relese. It's simply one tiny step forward in something that really didn't work before.